src/compute-client/src/command.proto

```diff
@@ -36,6 +36,16 @@ message ProtoComputeCommand {
         repeated mz_proto.ProtoU128 uuids = 1;
     }
 
+    message ProtoComputeStartupEpoch {
```

Comment:
nit: the current convention is to use nested message types (that is `Proto$V` nested inside `Proto$T`) for complex enum variants `$T::$V` (such as `ComputeCommand::CreateTimely`). 

Since `ProtoComputeStartupEpoch` corresponds to a proper Rust type (`mz_compute_client::command::ComputeStartupEpoch`) rather than a Rust variant, the corresponding `message ProtoComputeStartupEpoch` definition should probably go one level up (directly in `command.proto`).